### PR TITLE
feat: responsive img, toast and text utility added

### PR DESCRIPTION
**Pull request by ShreyashMahajan**
In this branch, three components are added.
1. Responsive image.
   i. Simple responsive image.
   ii. Round responsive image.
2. Toast/Snackbar is added.
3. Text utilities ( heading ).
i. Simple headings.
ii. Gray heading.
iii. Heading with the center.
please provide suggestion

### CODE REVIEW

File: componentLibrary.css [hunk 1]
```diff
@@ -497,4 +497,99 @@
...
+.heading-container {
+    width: 70%;
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    color: white;
+}
+
+.heading--one {
+    font-size: 48px;
```
Comment (Raghav888) [inline componentLibrary.css]: Please use rem

**Review by Raghav888 — APPROVED** [review]
Approved but in next PR change to rem

File: componentLibrary.css (hunk 1 above)
Comment (ShreyashMahajan) [reply componentLibrary.css]: oh sorry, I will change it to rem. Thank you.

**Review by Bharatrathod-81 — APPROVED** [review]
looks perfect

**Comment by ShreyashMahajan** [comment]
> looks perfect

thank you

**Pull request merged**